sequencing/vcf_to_plink/dxapp.json

```diff
@@ -83,22 +92,60 @@
     {
       "name": "fam_out",
       "label": "PLINK FAM file",
+      "help": "The FAM file generated by PLINK.",
       "class": "array:file",
       "patterns": [
         "*.fam"
       ]
     }
   ],
   "runSpec": {
-    "execDepends": [
-      {"name": "libgsl0ldbl"},
-      {"name": "pigz"},
-      {"name": "tabix"}
-    ],
     "interpreter": "bash",
-    "file": "src/vcf_to_plink.sh"
+    "file": "src/vcf_to_plink.sh",
+    "release": "14.04",
+    "distribution": "Ubuntu",
+    "timeoutPolicy": {
+      "main": {
+        "hours": 24
+      }
+    }
+  },
+  "regionalOptions": {
+    "aws:us-east-1": {
+      "assetDepends": [
+        {
+          "project": "project-FGfj3Qj035bYpjzg9y3bvXFG",
+          "folder": "/asset/",
+          "version": "0.0.1",
+          "name": "marylyn_asset"
+        }
+      ],
+      "systemRequirements": {
+        "*": {
+          "instanceType": "mem1_ssd1_x4"
+        }
+      }
+    },
+    "azure:westus": {
+      "assetDepends": [
+        {
+          "project": "project-FGp4k7j9k2Vy68736j2kBVPy",
+          "folder": "/",
+          "version": "0.0.1",
+          "name": "marylyn_asset"
+        }
```

Comment:
Please update record IDs to:
aws:us-east-1: record-FJ3zf8j08zVZFzqYG88Pbk7z
azure:westus: record-FJ4058j9Vf9ZBBVB2Q7b3VQz